modules/apps/oauth2-provider/oauth2-provider-test/src/testIntegration/java/com/liferay/oauth2/provider/rest/endpoint/access/token/grant/handler/test/JWTBearerGrantHandlerTest.java

```diff
@@ -62,7 +56,7 @@
  * @author Arthur Chan
  */
 @RunWith(Arquillian.class)
-public class JWTBearerGrantHandlerTest extends BaseGrantHandlerTestCase {
+public class JWTBearerGrantHandlerTest extends BaseClientTestCase {
```

Comment:
Hey @stian-sigvartsen 
It was my intend to create a separate base grant handler test class different from base client test class, because in idea, the authorization, token exchange, and resource requesting should all be separated in terms of testing. 
The base grant handler test class is for the part of token exchange.
But I think it's fine for now if we want to re-use the client test class.

Reply:
> The base grant handler test class is for the part of token exchange.
This is an interesting observation. We don't want to complicate other grant flow tests complexity if we don't have to, for sure.

So maybe something we could consider this again in the future. We'd need to apply a consistent approach to all grant flow testing in one PR.